src/Microsoft.Identity.Client/Platforms/net45/WindowsFormsWebAuthenticationDialogBase.cs

```diff
@@ -42,13 +42,8 @@ namespace Microsoft.Identity.Client
     [EditorBrowsable(EditorBrowsableState.Never)]
     public abstract class WindowsFormsWebAuthenticationDialogBase : Form
     {
-        private RequestContext RequestContext { get; }
+        private RequestContext RequestContext { get; set; } = new RequestContext(Guid.Empty);
```

Comment:
I noticed there is a ComVisible attribute on the class. Do you still want this to be internal? It will go against the ComVisible setting.